coordinator/coordinator.go

```diff
@@ -389,6 +389,7 @@ func (c *Coordinator) verifyCompletionOfSources(sources []metadata.NameVariant)
 			}
 		}
 		allReady = total == totalReady
+		time.Sleep(1 * time.Second)
```

Comment:
we need a figure out how to do this whole poll isready a bit better but that's a separate probz

Reply:
agreed